src/eventSubscription/tree/EventSubscriptionProvider.ts

```diff
@@ -66,7 +67,11 @@ export class EventSubscriptionProvider extends SubscriptionTreeItem {
         await wizard.prompt(actionContext);
         // tslint:disable-next-line:no-non-null-assertion
         showCreatingNode(wizardContext.newEventSubscriptionName!);
-        await wizard.execute(actionContext);
+        const message: string = localize('creatingEventSubscription', 'Creating event subscription "{0}"...', wizardContext.newEventSubscriptionName);
+        await vscode.window.withProgress({ title: message, location: vscode.ProgressLocation.Notification }, async () => {
+            // tslint:disable-next-line:no-non-null-assertion
+            await wizard.execute(actionContext!);
```

Comment:
Is there any confirmation that the event subscription finished creating?

Reply:
Yeah in `EventSubscriptionCreateStep.ts`